schema/collation_short_test_schema.json

```diff
@@ -0,0 +1,54 @@
+{"$schema": "https://json-schema.org/draft/2020-12/schema",
```

Comment:
Please make the JSON Schema be strict (not allow unspecified properties, etc). OK to make an explicit place to put unspecified properties, such as the free-form `attributes` array

Reply:
"additionalProperties": false,